doc/nrf/libraries/modem/location.rst

```diff
@@ -379,7 +379,10 @@ The following |NCS| applications and samples use this library:
 Limitations
 ***********
 
-* The Location library can only have one application registered at a time. If there is already an application handler registered, another initialization will override the existing handler.
+* The Location library can only have one application registered at a time.
+  If there is already an application handler registered, another initialization will override the existing handler.
+* GNSS API should not be used directly by the application when using the Location library.
+  Using the GNSS API from both the application and the Location library may lead to unexpected behavior.
```

Comment:
And what about the dynamics mode?

So it's not required by Location library that uses single fixes. Is this clear for the user?
And is it clear that dynamics mode is not helpful for single fix mode (non-continuous)?

Reply:
There are a lot of bells and whistles in the GNSS API and we can't explicitly list everything here. I would rather just say, that you shouldn't mix Location library and GNSS API usage. If you do so, you do so at your own risk.

I'll discuss with Tuomas if we should add a generic comment for dynamics mode regarding the GNSS operational mode.

Reply:
I'm not saying these things should be documented here, but I feel we should mention these topics somewhere.
Location library using single fixes could be in this document (I don't know where else would it be) and dynamics mode not being helpful in single fix mode could be documented into GNSS API.

Reply:
The dynamics mode does have some effect also in non-continuous modes. Thus, there's no need to update the dynamics mode documentation, I think.

Using GNSS API directly to configure dynamics mode with the Location lib is another thing. Personally, I would like to discourage GNSS API usage with the Location library, because that can lead to problems. Instead of providing instructions which do exactly that, we could consider adding support for configuring the dynamics mode in the Location library. @hiltunent, any opinions?

Reply:
This is all a bit confusing and contradicting what we discussed last week. And I wouldn't like to add a parameter which is close to useless. But it seems we are unwilling to instruct not to use dynamics mode in non-continuous mode either...
Just leaving things as they are (unclear raising questions in user's heads) is an option but not an ideal one either as someone else will try the same mistakes. Or will try to find a way to use dynamics mode thinking it will improve their accuracy data a lot.